packages/draft-js-export-markdown/src/stateToMarkdown.js

```diff
@@ -218,13 +218,20 @@ class MarkupGenerator {
             if (blockType === BLOCK_TYPE.CODE) {
               return content;
             }
+
+            const contentLength = content.length;
+            content = content.trimLeft();
+            const leftPad = contentLength - content.length;
+            content = content.trimRight();
+            const rightPad = contentLength - content.length - leftPad;
+
             // NOTE: We attempt some basic character escaping here, although
             // I don't know if escape sequences are really valid in markdown,
             // there's not a canonical spec to lean on.
             if (style.has(CODE)) {
               return '`' + encodeCode(content) + '`';
             }
-            content = encodeContent(text);
+            content = encodeContent(text.trim());
```

Comment:
Ran into some formatting issues with this if the trimmed result was a blank string, had to wrap the style logic in an additional length check. Also since the start / end are already trimmed above switched this to use `content` instead of `text`.

Reply:
Good catch!
```suggestion
            content = encodeContent(content);
```